modules/apigee_edge_apidocs/src/ApiDocListBuilder.php

```diff
@@ -82,4 +82,22 @@ public function buildRow(EntityInterface $entity) {
     return $row + parent::buildRow($entity);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getOperations(EntityInterface $entity) {
+    $operations = parent::getOperations($entity);
+
+    // Add "Re-import OpenAPI spec" link.
+    if ($entity->access('update') && $entity->hasLinkTemplate('reimport-spec-form')) {
```

Comment:
Because of [this](https://github.com/apigee/apigee-edge-drupal/blob/b4d0ba7d54ee404255dece1fa04afa3829858d97/modules/apigee_edge_apidocs/src/ApiDocAccessControlHandler.php#L50-L52) access check needs to be changed here.

Reply:
Changing to `$entity->access('reimport')`.